tests/test_quoteengine.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+from unittest.mock import patch
+from unittest.mock import MagicMock
+
+import pytest
+
+from tululbot.utils import QuoteEngine
+
+
+class FakeResponse:
+    def get_json(self):
+        return self.json
+
+    pass
+
+
+@pytest.fixture
+def qe():
+    return QuoteEngine()
+
+
+def cdn_with_branch(branch):
+    return 'https://cdn.rawgit.com/tulul/tulul-quotes/{}/quote.yaml'.format(branch)  # noqa
+
+
+def test_refresh_time(qe):
+    with patch('tululbot.utils.TimeHelper') as mock_dt:
+        with patch('tululbot.utils.requests') as mock_requests:
+            response1 = FakeResponse()
+            response1.status_code = 200
+            response1.json = {
+                "commit": {
+                    "sha": "beefbeef"
+                }
+            }
+
+            response1.ok = True
+
+            mock_requests.get.return_value = response1
+
+            qe._refresh_interval = 5

```

Comment:
changing this into instance variable is probably a good idea. I can't see why refresh interval should be private. or do you have reason for this?